### ISSUE TITLE
Collision/sewing error, Blender 2.91.0

### ISSUE THREAD

**Issue by JoshCodeStuff** [the3dadvantage/MC_29#1]
Steps:
1: Added character (MB-Lab)
2: Turned on collision while selecting character mesh
3: added plane
4: turned on cloth (in the MC menu)
5: sized plane and in edit mode duplicated it to the back
6: selected vertices I wanted to sew and clicked sewing lines
7: This error is thrown

Attached a picture of my plane setup

```
Python: Traceback (most recent call last):
  File "C:\Users\LtPinecone\AppData\Roaming\Blender Foundation\Blender\2.91\scripts\addons\MC_29-main\MC_29.py", line 4449, in execute
    cloth = MC_data['cloths'][ob['MC_cloth_id']]
KeyError: 'bpy_struct[key]: key "MC_cloth_id" not found'

location: <unknown location>:-1

```

![image](https://user-images.githubusercontent.com/49174146/102918707-702f7780-4455-11eb-84c4-43d2fdb8a1ee.png)

**Comment by JoshCodeStuff** [comment]
Blend file:
[Test1.zip](https://github.com/the3dadvantage/MC_29/files/5731629/Test1.zip)

**Comment by Ceztio** [comment]
have you tried to join those 2 planes together? 
![image](https://user-images.githubusercontent.com/35384866/104773098-ca7dd880-5774-11eb-8192-f26f9af024f3.png)

**Comment by JoshCodeStuff** [comment]
> have you tried to join those 2 planes together?
> ![image](https://user-images.githubusercontent.com/35384866/104773098-ca7dd880-5774-11eb-8192-f26f9af024f3.png)

Yes. To no avail. I have not tried in awhile so I will give it another try.

**Comment by NorthMountGames** [comment]
Click "cloth" again in edit mode.
Select vertices using edge. (or use either method. )
Finally, click on "sew lines"

**Comment by NorthMountGames** [comment]
> Click "cloth" again in edit mode.
> Select vertices using edge. (or use either method. )
> Finally, click on "sew lines"

I found this by accident, and I would say it's a complicated way ...

**Comment by JoshCodeStuff** [comment]
> Click "cloth" again in edit mode.

This seems to crash The latest Blender Version.
